packages/replay/src/session/getSession.ts

```diff
@@ -36,8 +36,9 @@ export function getSession({
 
     if (!isExpired) {
       return { type: 'saved', session };
-    } else if (session.sampled === 'error') {
-      // Error samples should not be re-created when expired, but instead we stop when the replay is done
+    } else if (!session.shouldRefresh) {
```

Comment:
The only thing, just to make this sure and clear, is that this means that when an error session expires without an error happened, it could be re-started as a session-session, right? So we may sample it and start sending data once the session resumes. IMHO that's OK, just wanting to be explicit in behavior!